index.js

```diff
@@ -221,15 +223,12 @@ class ResultStore {
         }
 
         // logging results
-        if (obj.emit) this.conn.loginfo(plugin, result.human);  // by request
+        if (obj.emit  && result.human.trim().length > 0) this.conn.loginfo(plugin, result.human);  // by request
         if (obj.err) {
             // Handle error objects by logging the message
-            if (util.isError(obj.err)) {
-                this.conn.logerror(plugin, obj.err.message);
-            }
-            else {
-                this.conn.logerror(plugin, obj.err);
-            }
+            const level = obj.emit_log_level ? obj.emit_log_level : 'error';
```

Comment:
This name of `emit_log_level` is misleading. When emit is specified, the result is logged at loginfo, as statically defined a few lines above. If one were to specify an `emit_log_level`, wouldn't they expect the level of the `emit: true` to change? 

This should be named differently, perhaps as `err_log_level`.